src/components/JobModalFields.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+export default (props) => {
```

Comment:
don't pass `props` - destructure the individual props that you're using so we have more control and we're not hiding props